src/index.ts

```diff
@@ -1,11 +1,5 @@
 export { Passage } from './classes/Passage/Passage';
-export * from './types/AuthStrategy';
-export * from './types/JWKS';
-export * from './types/MagicLink';
-export * from './types/PassageConfig';
-export * from './types/User';
 export * from './generated/models';
```

Comment:
You should be able to remove `export * from './generated/models;'. The idea is that any generated types that are returned by our classes have re-exported them from the `types.ts` file in each of the `classes/<ClassName>` directories.

Reply:
removed in [2bf3e5a](https://github.com/passageidentity/passage-node/pull/209/commits/2bf3e5a905137ac38fb0390560a30616c4552c53)